snap/snapcraft.yaml

```diff
@@ -75,8 +74,13 @@ parts:
       - "-DCMAKE_SKIP_RPATH=TRUE"
       - "-DCMAKE_PREFIX_PATH=$(echo $SNAPCRAFT_CMAKE_ARGS | awk -F= '{printf(\"%s/usr/lib/$SNAPCRAFT_ARCH_TRIPLET/cmake/Qt5\", $2)}')"
       - "--event-handlers=console_direct+"
+      - "--executor sequential"
+
     build-environment:
-      - IGNITION_VERSION: citadel
+      - IGNITION_VERSION: fortress
+      - GZ_VERSION: fortress
+      - PKG_CONFIG_PATH: ""
+      - MAKEFLAGS: " -j8"
```

Comment:
I think this is already being taken care of [here](https://github.com/snapcore/snapcraft/blob/9ad7a93ea880b560d3c8a364617c67a3b0b6d157/snapcraft/parts/plugins/colcon_plugin.py#L184).

Reply:
removed